Remotion/Web/Core/UI/Controls/WebButton.cs

```diff
@@ -134,25 +158,15 @@ private void Page_PreRenderComplete (object? sender, EventArgs e)
       _hasPagePreRenderCompleted = true;
     }
 
-    [MemberNotNull (nameof (_textWithHotkey))]
-    protected override void Render (HtmlTextWriter writer)
-    {
-      _textWithHotkey = HotkeyParser.Parse (Text);
-
-      base.Render (writer);
-    }
-
     protected override void AddAttributesToRender (HtmlTextWriter writer)
     {
-      if (string.IsNullOrEmpty (AccessKey) && _textWithHotkey!.Hotkey.HasValue) // TODO RM-8118: Debug assert _textWithHotkey not null
-        writer.AddAttribute (HtmlTextWriterAttribute.Accesskey, HotkeyFormatter.FormatHotkey (_textWithHotkey));
+      if (!string.IsNullOrEmpty (AccessKey))
+        writer.AddAttribute (HtmlTextWriterAttribute.Accesskey, AccessKey);
 
       if (IsLegacyButtonEnabled)
       {
-        if (!string.IsNullOrEmpty (_textWithHotkey!.Text))
-          Text = _textWithHotkey.Text;
-        else if (_icon != null && _icon.HasRenderingInformation)
-          Text = _icon.AlternateText;
+        if (Text.IsEmpty && _icon != null && _icon.HasRenderingInformation)
+          base.Text = _icon.AlternateText;
```

Comment:
I think we need to move this down to the base-call of AddAttributesToRender.

Before, it worked because we just always set base.Text. Now, it's at least cleaner (and possibly incorrect otherwise) if we don't make this the else-branch for assigning the base.Text value.